### DISCUSSION TITLE
How to obtain the metabolite concentration of a community

### DISCUSSION THREAD

**Discussion by clauSilvaA** [micom-dev/micom#135] [Q&A] [answered]
Hi!
I use micom to create my own community. I want to obtain the final concentration of a particular metabolite (butyrate for example) from the result.  
Can you help me, please?
Thanks for an excellent program.

**Comment by clauSilvaA** [comment]
I should add up the results of the metabolite flow using micom.measures.production_rates and  micom.measures.consumption_rates?

**Reply by cdiener** [reply, marked as answer]
So you won't get concentrations because metabolic models only predict fluxes. The questions is whether you want the net flux or the secretion flux. For instance, for butyrate we usually use the secretion flux because the host/gut has access to that pool. In that case it would just be the production_rates function you already found and filtering for butyrate in that list.

**Reply by clauSilvaA** [reply]
thank you very much!

**Reply by clauSilvaA** [reply]
but, in which unity is predicted fluxes?
